frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -45,7 +45,7 @@ frappe.views.QueryReport = Class.extend({
 		<div class="no-report-area msg-box no-border" style="display: none;"></div>\
 		<div class="chart_area" style="border-bottom: 1px solid #d1d8dd; padding: 0px 5%"></div>\
 		<div class="results" style="display: none;">\
-			<div class="result-area" style="height:400px;"></div>\
+			<div class="result-area" style="height:650px;"></div>\
```

Comment:
Try `70vh`

Reply:
Made the change @netchampfaris  :D Thanks!